README.Rmd

```diff
@@ -40,13 +41,14 @@ by using `emmeans`.
 **Scope:**
 
 * Continuous responses with normal (but potentially heteroscedastic) residuals.
-* Only marginal linear models without individual-level random effects are supported
+* Individual-level random effects are not supported, the focus lies on marginal linear models.
```

Comment:
can we switch the order? like "Marginal linear models (i.e. random effects are not supported)."

Reply:
done